packages/react-instantsearch-core/src/connectors/__tests__/connectRefinementList.js

```diff
@@ -388,16 +412,25 @@ describe('connectRefinementList', () => {
         another: { searchState: 'searchState' },
       });
 
-      searchState = cleanUp({ attribute: 'name2' }, searchState);
+      searchState = connect.cleanUp(
+        { attribute: 'name2', contextValue },
+        searchState
+      );
       expect(searchState).toEqual({
         refinementList: {},
         another: { searchState: 'searchState' },
       });
     });
 
     it('calling searchForItems return the right searchForItems parameters with limit', () => {
-      const parameters = searchForFacetValues(
-        { attribute: 'ok', limit: 15, showMoreLimit: 25, showMore: false },
+      const parameters = connect.searchForFacetValues(
+        {
+          attribute: 'ok',
+          limit: 15,
+          showMoreLimit: 25,
+          showMore: false,
+          contextValue,
```

Comment:
It's useless in that case, we don't use the context value inside the `searchForFacetValues`.